wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -296,7 +296,7 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
             return false
         }
 
-        return context.isTimeIntervalSinceLastMessageOneMinute || !context.isSameSenderAsPrevious || message.updatedAt != nil
+        return context.isTimeIntervalSinceLastMessageOneMinute || !context.isSameSenderAsPrevious || context.previousMessageIsKnock || message.updatedAt != nil || isBurstTimestampVisible(in: context)
```

Comment:
Why are you adding these?

Reply:
Cleaned it up since we already discussed it during our call.